source/common/network/utility.h

```diff
@@ -121,6 +121,14 @@ class Utility {
    */
   static bool urlIsUnixScheme(absl::string_view url);
 
+  /**
+   * Build a TCP URL
```

Comment:
Can you include some documentation here that this function handles formatting IPv6?